# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/aubio/hotfix-ffmpeg7.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- aubio-0.4.9/src/io/source_avcodec.c.vanilla	2025-11-09 17:56:04.357438652 +0100
+++ aubio-0.4.9/src/io/source_avcodec.c	2025-11-09 17:56:15.667167290 +0100
@@ -320,8 +320,8 @@
     SwrContext *avr = swr_alloc();
 #endif /* HAVE_AVRESAMPLE || HAVE_SWRESAMPLE */
 
-    av_opt_set_int(avr, "in_channel_layout",  input_layout,              0);
-    av_opt_set_int(avr, "out_channel_layout", output_layout,             0);
+    av_opt_set_int(avr, "in_channel",  input_layout,              0);
+    av_opt_set_int(avr, "out_channel", output_layout,             0);
     av_opt_set_int(avr, "in_sample_rate",     s->input_samplerate,       0);
     av_opt_set_int(avr, "out_sample_rate",    s->samplerate,             0);
     av_opt_set_int(avr, "in_sample_fmt",      s->avCodecCtx->sample_fmt, 0);
